frontend/libs/studio-components/src/components/StudioCombobox/StudioCombobox.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { forwardRef } from 'react';
+import { Combobox } from '@digdir/designsystemet-react';
+//TODO: Update import path when v1 of the Design system has been updated
```

Comment:
should we make a tech-debt issue for this in Github? 😊

Reply:
Sure! :) I've created an issue: https://github.com/Altinn/altinn-studio/issues/13531